bin/ch-test

```diff
@@ -212,11 +213,13 @@ test_examples () {
     echo 'executing example phase tests ...'
     for i in $examples; do
         if (test_make_check_image "$i"); then
+            bat_file=$(dirname "$i")/test.bats
             tag=$("${CHTEST_DIR}/chtest-make-auto" tag "$i")
-            export CH_TEST_TAG=$tag
-            bats "$(dirname "$i")/test.bats"
+            cp "$bat_file" "${tmp_}/${tag}_example.bats"
+            sed -i "s@# ch-test-set: ch_tag@export CH_TEST_TAG=$tag@g" "${tmp_}/${tag}_example.bats"
```

Comment:
Probably a much better way to do this. Thoughts?

Reply:
The syntax I'd suggest is:

```
CH_TEST_TAG=%ch_test_tag%

load "${CHTEST_DIR}/common.bash"
```

Then just substitute what's between the percents. That way, it's more clear that there is a variable substitution happening, and it will explode more clearly if it doesn't happen.

Unrelated, I think the idiom for sed is to use slash unless the regex or substitution contains slashes.

I also think we should make `CH_TEST_TAG` lower case, but that seems like a different PR.